modules/menu/menu.c

```diff
@@ -730,6 +730,7 @@ static void event_handler(enum ua_event ev, struct bevent *event, void *arg)
 			return;
 		}
 
+		++count;
 		bevent_stop(event);
 		break;
```

Comment:
Please could you try to replace the break by a return?

Increment count is not needed. It counts the numer of "real" calls. With SIPSESS_CONN the call is not yet allocated.

Reply:
In line `704` we count the number of calls. Right above this in line `726` we called `ua_accept` **successfully**. So we know our call count is no longer correct and must be incremented. That was my reasoning for incrementing the call count here.

Every other event handled here (except for **one** - `UA_EVENT_UNREGISTERING`) does not exit early either. So I thought the handling of `UA_EVENT_SIPSESS_CONN` should be fixed to not break the general `event_handler` functionality either, just like every other event. Further, `bevent_stop` is called, so this fix seemed pretty safe.

If you want me to put a `return` there instead, I will.

Reply:
Processing of `UA_EVENT_SIPSESS_CONN` should end after the call of `ua_accept()`.
Yes, `return` is correct.